include/amici/defines.h

```diff
@@ -19,6 +19,7 @@ constexpr double pi = 3.14159265358979323846;
 
 /* Return codes */
 #define AMICI_RECOVERABLE_ERROR          1
+#define AMICI_UNRECOVERABLE_ERROR       -1
```

Comment:
Is there a specific reason to have AMICI_UNRECOVERABLE_ERROR and AMICI_TOO_MUCH_WORK indistinguishable?

Reply:
no, value of `AMICI_UNRECOVERABLE_ERROR` does not matter, just needs to be strictly negative.